v63002/u_inref/gtm8694.csh

```diff
@@ -239,5 +240,26 @@ foreach restrict (`cat restrictlist.txt`)
 		endif
 	endif
 end
-
+echo ""
+echo "#----------------------------------------------------------------------"
+echo ""
+echo "TESTING A RANDOM FUNCTION WHEN THERE IS A SYNTAX ERROR IN RESTRICT.TXT AND CHECKING THE SYSLOG"
+echo ""
+rm $ydb_dist/restrict.txt
+cat > $ydb_dist/restrict.txt << EOF
+syntax error
+EOF
+chmod -w $ydb_dist/restrict.txt
+$gtm_tst/com/lsminusl.csh $ydb_dist/restrict.txt | $tst_awk '{print $1,$9}'
+$MUPIP trigger -triggerfile=trigger.trg >>& trigger.txt
+set rand='$'`$gtm_tst/com/genrandnumbers.csh 1 1 7`
+set fn=`$tst_awk "{print $rand}" fnlist.txt`
+set t1 = `date +"%b %e %H:%M:%S"`
+$ydb_dist/mumps -run $fn^gtm8694>&output.txt
+set t2 = `date +"%b %e %H:%M:%S"`
+# Included to avoid the syntax error in the syslog created when calling getoper (which invokes an m function)
```

Comment:
I did not understand this comment. Seems out of place.

Reply:
There would be extra syntax errors in the syslog because getoper invokes m functions. To avoid this, i did a time interval (the sleep 1 prevents the getoper syslog messages from occurring in the same second as t2)

Reply:
In that case, fix the restrict.txt syntax error before invoking getoper.csh. That way we keep the syntax error only for the mumps -run $fn^gtm8694 invocation.